handsome-hedgehogs/Bangazon-Site
# Js images

## Status
**READY**

## Description
User can upload a picture when creating a new product

## Steps to Test or Reproduce
```
1. git fetch
2. git checkout js-images
3. log in and navigate to sell something new
4. choose a picture on your local machine
6. complete the form
2. confirm that the image has been added to the wwwroot/products folder
3. confirm the product was created
```

## Link to Feature Ticket
#14
